stats/Record.md

```diff
@@ -51,7 +51,10 @@ contextual information of an exemplar, for example trace id, span id or dropped
 ## Recording Stats
 
 Users should record Measurements against a context, either an explicit context or the implicit 
-current context. Tags from the context are recorded with the Measurements if they are any.
+current context. Tags from the context are recorded with the Measurements if they are any. 
```

Comment:
Please change the example to show this as well.

Reply:
Done.